tests/test_import.py

```diff
@@ -0,0 +1,28 @@
+# This file is part of the_monster.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import lsst.the.monster.testImport
+
+
+class TestImport(unittest.TestCase):
+    def test_import(self):
+        self.assertTrue(hasattr(lsst.the.monster.testImport, 'testImport'))
```

Comment:
I don't think we need the full testImport file.  This could just check that the version exists (post-installation with scons it will have a `__version__` that you can look at).

Reply:
I think this has been fixecd